component/s3storage/client_test.go

```diff
@@ -503,18 +504,19 @@ func (s *clientTestSuite) TestListBuckets() {
 	s.assert.Contains(buckets, storageTestConfigurationParameters.BucketName)
 }
 
-func (s *clientTestSuite) TestDefaultBucketName() {
-	defer s.cleanupTest()
-	// write config with no bucket name
-	config := fmt.Sprintf("s3storage:\n  key-id: %s\n  secret-key: %s\n  endpoint: %s\n  region: %s\n  use-path-style: %t\n",
-		storageTestConfigurationParameters.KeyID, storageTestConfigurationParameters.SecretKey,
-		storageTestConfigurationParameters.Endpoint, storageTestConfigurationParameters.Region,
-		storageTestConfigurationParameters.UsePathStyle)
-	err := s.setupTestHelper(config, false)
-	s.assert.NoError(err)
-	buckets, _ := s.client.ListBuckets()
-	s.assert.Equal(buckets[0], s.client.Config.authConfig.BucketName)
-}
+// TODO: Cannot run this test in current test account with buckets we don't have permission for
```

Comment:
Is there a way to fix this? Do we need to have the CI credentials only work for one bucket?

Reply:
The default bucket is the first bucket listed. Since there are other available buckets, then this won't work as we don't have access permissions for the first bucket. So we would need a separate account with just our buckets in it.